pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -124,34 +164,57 @@ class TelegramHandler(EventHandler):
     def __init__(self, bot, config):
         self.bot = bot
         self.tbot = None
-        self.master = config.get('master', None)
+        master = config.get('master', None)
         self.pokemons = config.get('alert_catch', {})
         self.whoami = "TelegramHandler"
         self.config = config
+        if master == None:
+            self.master = None
+            return
+
+        with self.bot.database as conn:
+            # if master is not numeric, try to fetch it from the database
+            if not re.match(r'^[0-9]+$', str(master)): # master is numeric
```

Comment:
Regarding code readability... This is what the comments in the code are for. Explicit is better than implicit. :D

Though it is not worth bothering with from my opinion ("readability" is a very personal criterium at this level), I can change it when working on #5192 if you insist (because it is not worth wasting so much time in discussions either). Which is waiting for this PR to be merged because it will base on it.